lib/src/types.rs

```diff
@@ -62,20 +62,20 @@ pub enum Stmt {
     TypedStmt(Box<Stmt>, Type),
 }
 
-#[derive(Debug, Clone, Deserialize, Serialize)]
+#[derive(Debug, Clone, Deserialize, Serialize, PartialEq)]
```

Comment:
If we're gonna derive PartialEq we might as well derive Eq

Reply:
Why do we need it? We can add it in the future when we need it.